data/devices/nokia.yml

```diff
@@ -0,0 +1,33 @@
+- name: Nokia XL
+  id: normandy
+  codenames:
+    - normandy
+  architecture: armeabi-v7a
+
+   system:
```

Comment:
This entire section (`system`, `cache`, `data`, `boot`, `recovery`) is indented one space too far. The build failed with:

```
nokia.yml: Failed to convert file: yaml-cpp: error at line 7, column 10: illegal map value
```

You can check the syntax at: http://www.yamllint.com/